InTheMiddle2/InTheMiddle2/ViewController.swift

```diff
@@ -27,27 +44,33 @@ class ViewController: UIViewController, UISearchBarDelegate{
         minimalStyle = try! GMSMapStyle.init(contentsOfFileURL: Bundle.main.url(forResource: "modernminimal", withExtension: "json")!)
         mapView.mapStyle = minimalStyle
         
+        self.googleMapsView?.camera = camera
```

Comment:
The google map works perfectly and displays the markers from point to point. Greatjob